notebooks/nzh-20230417-00-embeddings-classifier.ipynb

```diff
@@ -0,0 +1,700 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.                         (1 / (1 + f.exp(-1 * explode_preds.probability))).alias("probability"))
</pre><p>You should be able to use the pre-computed confidence field, which should be the sigmoid of this probability field.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I don't think there is a confidence field in the predictions map. There is only rank, index, label, mapped_label, probability

Reply:
And if I take the sigmoid of the sigmoid of the probabilities, all my values are between 0.5 and 0.8 (there is bar graph in the notebook). Not sure if this was correct since that would mean all tracks contain bird calls?

Reply:
I must have mixed this up with the v1 dataset, this is fine then.